README.md

```diff
@@ -101,6 +101,39 @@ If your environment requires a proxy, define the proxy properties in the JSON fi
   "proxy": "<proxy_host>:<proxy_port>"
 ```
 
+## Exception handling
+
+All the exceptions raised by the OneView inherits from HPOneViewException.
```

Comment:
Change: OneView for OneView Python SDK  - is the sdk that raises the exceptions;
inherits for inherit